TreasureHunt/Assets/2.Scripts/Hunter/FieldView/TreasureSetupController.cs

```diff
@@ -0,0 +1,137 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using SimpleJSON;
+
+public class TreasureSetupController : MonoBehaviour {
+
+	public static TreasureSetupController instance = null;
+
+	void Awake()
+	{
+		if (instance == null)
+
+			instance = this;
+
+		else if (instance != this)
+
+			Destroy(gameObject);
+
+		DontDestroyOnLoad (gameObject);
+
+	}
+
+	void Start(){
```

Comment:
Your brace style differs from Hyun's. You guys need to get together and figure out what your coding convention is :).